CandidateExercise/ConfigurationExcerciseTests/ConfigTests.cs

```diff
@@ -1,34 +1,101 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.Linq;
 using log4net;
 using log4net.Config;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 using Moq;
 
+
 namespace FutureWonder.Exercises.Configuration
 {
     using KVPList = IList<KeyValuePair<string, ConfigValue>>;
-
+    using KVP = KeyValuePair<string, ConfigValue>;
+    using KList = IList<string>;
     [TestClass]
     public class ConfigTests
     {
+        #region variables
+
         private readonly ILog _log = LogManager.GetLogger(nameof(ConfigTests));
         private readonly MockRepository _repository = new MockRepository(MockBehavior.Loose);
         private Mock<IPersistSource> _persistSource;
         private Config _config;
 
+        private List<String> keyList { get { return new List<string>() { "Key1", "Key2", "Key3" }; } }
+        private KVPList kvplist
```

Comment:
We prefer to use C# naming conventions to help minimize confusion where possible. For Properties, please capitalize the first letter (even if private, I believe).

Is there a reason you chose to use get-only Properties instead of read-only Variables?

Reply:
By using get-only properties I was able to ref the properties to build up other complex objects otherwise I was getting the error  "A field initializer cannot reference the non-static field, method, or property" when using read-only property